src/components/notebook-utils.js

```diff
@@ -68,9 +68,13 @@ export const analysisNameInput = ({ inputProps, ...props }) => h(ValidatedInput,
   }
 })
 
-export const tools = { rstudio: { label: 'RStudio', ext: 'Rmd' }, jupyter: { label: 'Jupyter', ext: 'ipynb' } }
+export const tools = {
+  RStudio: { label: 'RStudio', ext: 'Rmd', imageIds: ['terra-jupyter-bioconductor', 'RStudio', 'terra-jupyter-bioconductor_legacy'], defaultImageId: 'terra-jupyter-bioconductor' },
+  Jupyter: { label: 'Jupyter', ext: 'ipynb', imageIds: ['terra-jupyter-hail', 'terra-jupyter-python', 'terra-jupyter-gatk', 'Pegasus', 'terra-jupyter-gatk_legacy'], defaultImageId: 'terra-jupyter-gatk' },
+  galaxy: { label: 'galaxy' }
+}
```

Comment:
Curious why casing is different for `RStudio`, `Rmd`, `Jupyter` and `Pegasus` vs. the other keys and values in `tools` object

Reply:
The casing is directly tied to the existing `tool` labels we use for runtimes. The strings are re-used here, matching the existing capitalization